leader-worker-set/rockcraft.yaml

```diff
@@ -0,0 +1,52 @@
+# Source https://github.com/kubernetes-sigs/lws/blob/v0.7.0/Dockerfile
+name: leader-worker-set
+summary: An image for LeaderWorkerSet
+description: |
+  LeaderWorkerSet: An API for deploying a group of pods as a unit of replication.
+  It aims to address common deployment patterns of AI/ML inference workloads,
+  especially multi-host inference workloads where the LLM will be sharded and run across multiple devices on multiple nodes.
+version: "0.7.0"
+license: Apache-2.0
+base: ubuntu@22.04
+
+platforms:
+  amd64:
+
+services:
+  kubeflow-trainer:
+    override: merge
+    command: "/manager"
+    startup: enabled
+
+run-user: _daemon_
+
+parts:
+  security-team-requirement:
+    plugin: nil
+    override-build: |
+      mkdir -p ${CRAFT_PART_INSTALL}/usr/share/rocks
+      (echo "# os-release" && cat /etc/os-release && echo "# dpkg-query" && \
+       dpkg-query -f '${db:Status-Abbrev},${binary:Package},${Version},${source:Package},${Source:Version}\n' -W) > \
+       ${CRAFT_PART_INSTALL}/usr/share/rocks/dpkg.query
+
+  kubeflow-trainer:
+    plugin: go
+    build-snaps:
+      - go/1.24/stable
+    source: https://github.com/kubernetes-sigs/lws
+    source-type: git
+    source-tag: v0.7.0
+    source-depth: 1
+    source-subdir: cmd/
+    build-environment:
+      - CGO_ENABLED: 0
+      - GOOS: linux
+    override-build: |
+      # cd into $CRAFT_PART_SRC_WORK due to https://github.com/canonical/craft-parts/issues/427
+      cd $CRAFT_PART_SRC_WORK
+      go mod download
+      go build -a -o $CRAFT_PART_INSTALL/manager main.go
```

Comment:
suggestion (non-blocking): what if we instead run [the same instruction as in the upstream Dockerfile](https://github.com/kubernetes-sigs/lws/blob/v0.7.0/Dockerfile#L18), with `make build`, so that it's easier for future developers to map changes to this file to changes to the Dockerfile? Otherwise it could be harder to unravel what to update in the future